script.js

```diff
@@ -28,4 +50,31 @@ function agendar() {
     )
   );
   console.log(agenda);
+  for (let index = 0; index < agenda.length; index++) {
+    document.getElementById("f").innerHTML = "Fecha: " + agenda[index].fecha;
+    document.getElementById("d").innerHTML =
+      "Desayuno: " + agenda[index].desayuno;
+    document.getElementById("a").innerHTML =
+      "Almuerzo: " + agenda[index].almuerzo;
+    document.getElementById("m").innerHTML =
+      "Merienda: " + agenda[index].merienda;
+    document.getElementById("c").innerHTML = "Cena: " + agenda[index].cena;
+  }

```

Comment:
- Una forma mas prolija de escribir un for (cuando no se necesita el indice para alguna utilidad en especial) es escribirlo de la forma:
```
for ( var dia of agenda ) {
document.getElementById("f").innerHTML = "Fecha: " + dia.fecha;
    document.getElementById("d").innerHTML =
      "Desayuno: " + dia.desayuno;
    document.getElementById("a").innerHTML =
      "Almuerzo: " + dia.almuerzo;
    document.getElementById("m").innerHTML =
      "Merienda: " + dia.merienda;
    document.getElementById("c").innerHTML = "Cena: " + dia.cena;
}
```